website/content/docs/operators/configuration/config-intro.md

```diff
@@ -1,17 +1,18 @@
 ---
-title: Intro to Heron Configuration
+title: Intro to Heron Internal Configuration
 ---
 
-Heron can be configured at two levels:
+Heron internal configuration can be configured at two levels:
```

Comment:
What do you mean by "internal" here? Make sure to clarify because there are a lot of moving pieces at work in Heron.